utils/mosaic_processor.py

```
@@ -33,17 +33,72 @@
+        # Build command to run monitoring script in new window
+        cmd = [
+            "cmd", "/c", "start",
+            "Mosaic Processor Progress",  # Window title
+            python_exe, str(monitor_script),
+            "--status-file", str(status_file_path),
+            "--watch"
+        ]
+
+        # Launch the process
+        creation_flags = getattr(subprocess, "CREATE_NEW_CONSOLE", 0)
+        process = subprocess.Popen(
```

Ruff (0.14.2)

`subprocess` call: check for execution of untrusted input

(S603)